Tests/HummingbirdTests/HTTPTests.swift

```diff
@@ -89,6 +89,19 @@ class HTTPTests: XCTestCase {
         XCTAssert(HBMediaType(from: "audio/ogg")?.isType(.audioOgg) == true)
     }
 
+    func testMediaTypeMatching() {
```

Comment:
We should also add `mismatching` tests

Reply:
Including `application/json` not case matching on `application/json; charset=utf8` (since the latter is more explicit). And mismatching when the type is different entirely.